aten/src/ATen/FunctionalStorageImpl.cpp

```diff
@@ -89,10 +90,23 @@ bool Alias::apply_updates() {
   return any_updates;
 }
 
+c10::SymInt get_nbytes(const Tensor& value) {
+  if (value.unsafeGetTensorImpl()->has_symbolic_sizes_strides()) {
+    // Today, the two implementations of SymInt are in Python (proxy tensor),
+    // and lazy tensor (LTC/XLA).
+    // LTC hasn't implemented SymInt support yet though (torch::lazy::SymIntNodeImpl).
+    // Once it does, we should remove this check.
+    if (value.key_set().has(c10::DispatchKey::Python)) {
+      return at::detail::computeStorageNbytesSymInt(value.sym_sizes(), value.sym_strides(), value.dtype().itemsize(), value.storage_offset());
```

Comment:
cc @Krovatkin I ended up doing this to fix LTC test failures. Just confirming that this seems ok to you?

Originally, I wrote this code to say "if the inner tensor is using symbolic strides, then do symint stuff".

That breaks LTC though, because none of the methods on `torch::lazy::SymIntNodeImpl` are implemented yet so it can't actually trace any symbolic shape stuff yet.

I figure that once LTC/XLA are ready to trace symbolic shapes, then we can fix this?